### DISCUSSION TITLE
Another feature request

### DISCUSSION THREAD

**Discussion by cvrle77** [muraliwebworld/push-notification-plugin-for-wordpress-websites#13] [Ideas]
If we could get 'time of subscription' or 'registered at', we could analyze our data in different ways and do something about it, find proper timing to send notifications, if subscribers stop incoming this could indicate a problem, etc etc...

![image](https://github.com/muraliwebworld/push-notification-plugin-for-wordpress-websites/assets/16167396/bd4c8c85-782d-4685-974b-f44331c79e55)

**Comment by muraliwebworld** [comment]
I will look into this and i will add it in next week that is next to next release because it requires db table changes, i need to add alter table code it will require more testing, before June 01 i will try to implement most of changes
